src/cloud-functions.ts

```diff
@@ -168,6 +168,7 @@ export interface TriggerAnnotated {
     httpsTrigger?: {};
     eventTrigger?: {
       eventType: string;
+      failurePolicy?: object;
```

Comment:
Curious as to why failurePolicy is inside the eventTrigger and not outside like the other options like timeout and memory? was that because it does not apply to http functions?

Reply:
This is per the Google cloud docs - since it only applies to event-triggered functions, its a property of EventTrigger: https://cloud.google.com/functions/docs/reference/rest/v1/projects.locations.functions#EventTrigger

Reply:
ah, makes sense